### PR TITLE
Update nf core publication

### PR DESCRIPTION
**Pull request by ewels**
Updated the reference in the template and tools readme for the new paper.

Too slow @drpatelh! 😉

### CODE REVIEW

File: nf_core/pipeline-template/{{cookiecutter.name_noslash}}/README.md [hunk 1]
```diff
@@ -67,6 +67,11 @@ For further information or help, don't hesitate to get in touch on [Slack](https
...
-You can cite the `nf-core` pre-print as follows:  
-
-> Ewels PA, Peltzer A, Fillinger S, Alneberg JA, Patel H, Wilm A, Garcia MU, Di Tommaso P, Nahnsen S. **nf-core: Community curated bioinformatics pipelines**. *bioRxiv*. 2019. p. 610741. [doi: 10.1101/610741](https://www.biorxiv.org/content/10.1101/610741v1).
+You can cite the `nf-core` pre-print as follows:
+
+> **The nf-core framework for community-curated bioinformatics pipelines**
+>
+> Philip Ewels, Alexander Peltzer, Sven Fillinger, Harshil Patel, Johannes Alneberg, Andreas Wilm, Maxime Ulysse Garcia, Paolo Di Tommaso & Sven Nahnsen
+>
+> [_Nat Biotechnolgy_ (2020)](https://doi.org/10.1038/s41587-020-0439-x);
```
Comment (drpatelh) [inline nf_core/pipeline-template/{{cookiecutter.name_noslash}}/README.md]: Typooooooooooooooo!!!!!!

File: README.md [hunk 2]
```diff
@@ -561,5 +561,11 @@ INFO: Successfully synchronised [n] pipelines
...
 
-If you use `nf-core tools` in your work, please cite the `nf-core` preprint as follows:  
-Ewels PA, Peltzer A, Fillinger S, Alneberg JA, Patel H, Wilm A, Garcia MU, Di Tommaso P, Nahnsen S. **nf-core: Community curated bioinformatics pipelines**. *bioRxiv*. 2019. p. 610741. [doi: 10.1101/610741](https://www.biorxiv.org/content/10.1101/610741v1).
+If you use `nf-core tools` in your work, please cite the `nf-core` preprint as follows:
+
+> **The nf-core framework for community-curated bioinformatics pipelines**
+>
+> Philip Ewels, Alexander Peltzer, Sven Fillinger, Harshil Patel, Johannes Alneberg, Andreas Wilm, Maxime Ulysse Garcia, Paolo Di Tommaso & Sven Nahnsen
+>
+> [_Nat Biotechnolgy_ (2020)](https://doi.org/10.1038/s41587-020-0439-x);
```
Comment (drpatelh) [inline README.md]: Typooooooooooooooo!!!!!!

Comment (ewels) [reply README.md]: 🤦‍♂

File: nf_core/pipeline-template/{{cookiecutter.name_noslash}}/README.md (hunk 1 above)
Comment (ewels) [reply nf_core/pipeline-template/{{cookiecutter.name_noslash}}/README.md]: 🤦‍♂

**Pull request merged**